docs/features/if_statements.md

```diff
@@ -0,0 +1,32 @@
+# If Statements
+
+```js
+// Simple if statement that will always be false
+
+if (1 /= 1) { // usually if (1 != 1) in other langugaes
+	print("This will never run")
+} else if (false) {
+	print("This will never run")
+} else {
+	print("This will always run")
+}
+
+// The condutional let will only run the block of code if the value can be assigned to the variable
+if let i:str = 1 {
```

Comment:
I think the type checker will get pissed by this

Reply:
Odd, i thought that it would be fine with it

Reply:
ah, i see, the syntax thing gets mad